### PR TITLE
Fix selection options

### LINKED ISSUE #4211: Some menu items and options in the navigation toolbar don't work
**Explain the bug**
Write a clear description of what the bug is.

**To reproduce**
1. Go to any folder
2. Open selection flyout, click on select all
3. Nothing happens
4. Repeat for all other options in flyout with same result

**Expected behavior**
The flyout items should perform their tasks.

**Logfile**
Nothing relevant. 

**Screenshots**

https://user-images.githubusercontent.com/59544401/111933877-8ca37600-8a7d-11eb-8e93-198d3d23c2e5.mp4

**System Information**
 - OS Version: 21337.1010
 - App version: 1.4.47.0

**Additional comment**
Restart does not fix.
The keyboard shortcuts, however, do still work.

### PR DESCRIPTION
**Pull request by d2dyno1**
**Resolved / Related Issues**

- Closes #4211

**Details of Changes**

Fixes selection commands binding. Can you confirm that everything is working correctly after this change? @winston-de

**Validation**

How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

### CODE REVIEW

**Comment by winston-de** [comment]
I'm testing it now. You may have seen it mentioned earlier, but new tab, new window, and new pane in the ... menu are also broken.

**Review by yaichenbaum — APPROVED** [review]

File: Files/Views/ModernShellPage.xaml [hunk 1]
```diff
@@ -248,9 +248,9 @@
                 x:FieldModifier="public"
                 
                 MultiselectEnabled="{x:Bind InteractionViewModel.MultiselectEnabled, Mode=TwoWay}"
-                SelectAllInvokedCommand="{x:Bind SelectAllContentPageItemsCommand}"
-                InvertSelectionInvokedCommand="{x:Bind InvertContentPageSelctionCommand}"
-                ClearSelectionInvokedCommand="{x:Bind ClearContentPageSelectionCommand}"
+                SelectAllInvokedCommand="{x:Bind SelectAllContentPageItemsCommand, Mode=OneWay}"
```
Comment (gave92) [inline Files/Views/ModernShellPage.xaml:251]: Curious: is this change necessary given the other changes e.g. `new RelayCommand(() => SlimContentPage?.SelectAllItems());`?

Comment (yaichenbaum) [reply Files/Views/ModernShellPage.xaml:251]: I think the default is set to `OneTime`.

Comment (d2dyno1) [reply Files/Views/ModernShellPage.xaml:251]: I've also noticed these were set somewhere else, I've added `Mode=OneWay` because it was like that before Interaction removal and to unify with rest of those. @gave92

Comment (gave92) [reply Files/Views/ModernShellPage.xaml:251]: Thanks @d2dyno1 that's fine by me.
@yair100 I asked because given the changes to ModernShellPage.xaml.cs I think OneTime should be ok here.

Comment (d2dyno1) [reply Files/Views/ModernShellPage.xaml:251]: @gave92 Not so sure about `OneTime` here because it'd be set to null on home page and then it wouldn't update upon navigation

Comment (gave92) [reply Files/Views/ModernShellPage.xaml:251]: I'd bet that when you do this, the "WorkingDirectory" value is evaluted each time the command is called, so it would update on navigation, but I'm actually not sure :)
```cs
public ICommand Cmd = new RelayCommand(() => NavigationHelpers.OpenDirectoryInTerminal(this.FilesystemViewModel.WorkingDirectory, this));
```
